flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/testutils/KafkaSinkExternalContext.java

```diff
@@ -100,10 +98,7 @@ private void createTopic(String topicName, int numPartitions, short replicationF
                 replicationFactor);
         NewTopic newTopic = new NewTopic(topicName, numPartitions, replicationFactor);
         try {
-            kafkaAdminClient
```

Comment:
The diff `KafkaSinkExternalContext` should go into the FLINK-29914 commit.

Reply:
Moved.